jobs/async-upload/job/config.py

```diff
@@ -27,6 +28,7 @@ def _parser() -> cap.ArgumentParser:
     p.add("--source-aws-endpoint")
     # OCI registry
     p.add("--source-oci-uri")
+    p.add("--source-oci-registry")
```

Comment:
This (along with the destination) are added because a `.dockerconfigjson` file may contain multiple credentials for each registry. So its better for the consumer to declare which registry they intend to upload to, to make credential parsing more straight forward